packages/server/src/database/knex/migrations/20200425004137_create-users.js

```diff
@@ -0,0 +1,29 @@
+
+exports.up = function(knex) {
+  return knex.schema.hasTable('UserAddresses').then(function(sequelizeExists) {
+    if (sequelizeExists) {
+      return knex.schema.renameTable('UserAddresses', 'Users').alterTable('Users', function (table) {
+        table.boolean('addressVerified').defaultTo(false)
+        table.renameColumn('userId', 'userEmailId')
+        table.dropForeign('userEmailId', 'UserAddresses_userId_fkey') // need to recreate constraint with set null on delete
+        table.foreign('userEmailId').references('UserEmails.id').onDelete('SET NULL')
+        table.datetime('createdAt').defaultTo(knex.fn.now()).alter()
+        table.datetime('updatedAt').defaultTo(knex.fn.now()).alter()
+      })
+    } else {
+      return knex.schema.createTable('Users', function (table) {
+        table.increments('id')
+        table.string('address').unique()
```

Comment:
It would be useful to have an index here, not sure if `unique` does it by default

Reply:
It does

Reply:
Added some extra indexes in other places (e.g. foreign keys, expires at)